puppet-lint.gemspec

```diff
@@ -32,5 +32,5 @@ Gem::Specification.new do |spec|
   ]
   spec.license = 'MIT'
 
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.7'.freeze)
+  spec.required_ruby_version = Gem::Requirement.new('>= 2.5'.freeze)
```

Comment:
Should this be in a separate PR, if we do decide to do this?

Reply:
It would enhance visibility I guess.

The reason I included it here was because I kind of see it as part of this change.

Happy either way though! :-)

Reply:
I don't think this downgrade should be in this PR

Reply:
See https://github.com/puppetlabs/puppet-lint/pull/85#issuecomment-1439164696

Gonna fix this up shortly